rosbag2_bag_v2_plugins/src/rosbag2_bag_v2_plugins/storage/rosbag_v2_storage.hpp

```diff
@@ -43,6 +43,10 @@ class RosbagV2Storage : public rosbag2_storage::storage_interfaces::ReadOnlyInte
 
   rosbag2_storage::BagMetadata get_metadata() override;
 
+  uint64_t get_bagfile_size() const override;
```

Comment:
Method doc?

Reply:
This method behaves exactly like how the document on the interface behaves.
Is this fine as is? Otherwise I'd just be copying in the documentation from `rosbag2_storage`.

On the other method, I was able to make it more specific so its kind of a different case.